### PR TITLE
feat: Added a Makefile with tasks to run Jekyll container locally

### PR DESCRIPTION
**Pull request by zusyed**
This PR adds a Makefile to the project with commands to install and update the dependencies and run the serve the site using Jekyll docker container.

# QA
- Run the commands in the Makefile and make sure they work. There are a few commands there but the most important one is `make start`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,19 +3,20 @@
 <span><img align="right" src="https://raw.githubusercontent.com/tophat/getting-started/master/branding/assets/top-hat-open-source-logo-diamond.png" alt="Logo"></span>
 
 [![All Contributors](https://img.shields.io/badge/all_contributors-5-orange.svg?style=flat)](#contributors)
-[![Slack workspace](https://slackinvite.dev.tophat.com/badge.svg)](https://opensource.tophat.com/slack)
```
Comment (zusyed) [inline README.md:6]: Removed the Slack badge and other Slack links from the README because the links don't go anywhere.

File: Gemfile.lock [hunk 2]
```diff
@@ -252,11 +252,11 @@ GEM
...
-  x86_64-darwin-19
+  ruby
 
 DEPENDENCIES
   github-pages (~> 209)
   minima (~> 2.5)
 
 BUNDLED WITH
-   2.2.1
+   2.1.4
```
Comment (mcataford) [inline Gemfile.lock:262]: Interesting, why the downgrade?

File: README.md [hunk 3]
```diff
@@ -3,19 +3,20 @@
...
 ## I want to Contribute
-The easiest way to get started is to join the [slack workspace](https://opensource.tophat.com/#join-slack) and head to the opensource-portal channel.
 
 ### Local Development
-This site uses Jekyll
- - Install Jekyll by following docs on [their website](https://jekyllrb.com/docs/)
-   - If you want to use Bundler to add Jekyll to your project, follow [this doc](https://jekyllrb.com/tutorials/using-jekyll-with-bundler/) instead
- - Run `bundle exec jekyll serve` to get the site running locally
+This site uses Jekyll. There are multiple ways to install and run Jekyll:
+ 1. The easiest way is using the Jekyll docker [image](https://hub.docker.com/r/jekyll/jekyll/). There is a [Makefile](./Makefile) in this repository which uses the docker image
```
Comment (mcataford) [inline README.md:14]: I would add an explicit mention that you need to have `docker` set up already, even though it's implicitly stated here.

**Review by mcataford — DISMISSED** [review]
This is pretty rad! Great work! 🚀

File: Gemfile.lock (hunk 2 above)
Comment (zusyed) [reply Gemfile.lock:262]: The latest Jekyll docker image uses this version of Bundler. I didn't notice any issue but it was giving a warning so downgraded it.

File: README.md (hunk 3 above)
Comment (zusyed) [reply README.md:14]: Good point. Updated to explicitly state to have Docker installed and running before.

Comment (zusyed) [reply README.md:14]: The above change removed your approval. Can you approve again?

**Review by AnvarGazizovTH — APPROVED** [review]

**Pull request merged**